doc/api/process.md

```diff
@@ -2632,6 +2635,7 @@ cases:
 [`EventEmitter`]: events.md#events_class_eventemitter
 [`NODE_OPTIONS`]: cli.md#cli_node_options_options
 [`Promise.race()`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/race
+[`Worker constructor`]: worker_threads.html#worker_threads_new_worker_filename_options
```

Comment:
```suggestion
worker_threads.md#worker_threads_new_worker_filename_options
```

The `md` gets changed to `html` during generation of the html docs. Keeping it as `md` here will both get rid of the linting issue and also make the link work both in the GitHub interface for this file. (This is a recent change to our docs generation, and it is possible we may have missed updating our docs-about-docs somewhere.)

Reply:
@Trott - Thanks for the review, I have followed your suggestion. Please have a look.